stepfunctions/src/main/scala/awscala/stepfunctions/StepFunctions.scala

```diff
@@ -0,0 +1,131 @@
+package awscala.stepfunctions
+
+import java.util
+import java.util.concurrent.TimeUnit
+
+import awscala._
+import com.amazonaws.ClientConfiguration
+import com.amazonaws.services.stepfunctions.model._
+import com.amazonaws.services.{ stepfunctions => aws }
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ blocking, ExecutionContext, Future }
+import scala.util.Try
+
+object StepFunctions {
+  private val defaultSocketTimeout = TimeUnit.SECONDS.toMillis(70).toInt
+
+  def apply(credentials: Credentials)(implicit region: Region): StepFunctions =
+    apply(BasicCredentialsProvider(credentials.getAWSAccessKeyId, credentials.getAWSSecretKey))(region)
+
+  def apply(accessKeyId: String, secretAccessKey: String)(implicit region: Region): StepFunctions =
+    apply(BasicCredentialsProvider(accessKeyId, secretAccessKey))(region)
+
+  def apply(credentialsProvider: CredentialsProvider = CredentialsLoader.load())(
+    implicit
+    region: Region = Region.default()): StepFunctions =
```

Comment:
Why do you have default value only for this method? I don't think we should have the default value here.

Reply:
This is the way it is in all of the other modules. Want me to change it or keep it consistent?

Reply:
Ah, sorry I misunderstood the code. Never mind.